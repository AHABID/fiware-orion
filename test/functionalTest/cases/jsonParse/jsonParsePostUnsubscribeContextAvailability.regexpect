1: ***********************************
HTTP/1.1 200 OK
Content-Length: 144
Content-Type: application/json
Date: REGEX(.*)

{
    "statusCode": {
        "code": "404", 
        "reasonPhrase": "No context element found"
    }, 
    "subscriptionId": "012345678901234567890123"
}
2: ***********************************
