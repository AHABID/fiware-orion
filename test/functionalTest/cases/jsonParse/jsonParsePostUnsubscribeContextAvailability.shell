echo "1: ***********************************"

url="/NGSI9/unsubscribeContextAvailability"
payload='{
  "subscriptionId": "012345678901234567890123"
}'
curlJson ${url} "${payload}"


echo "2: ***********************************"
