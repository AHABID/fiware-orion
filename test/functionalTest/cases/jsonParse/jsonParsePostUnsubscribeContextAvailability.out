1: ***********************************
HTTP/1.1 200 OK
Content-Length: 144
Content-Type: application/json
Date: Mon, 02 Jun 2014 08:20:51 GMT

{
    "statusCode": {
        "code": "404", 
        "reasonPhrase": "No context element found"
    }, 
    "subscriptionId": "012345678901234567890123"
}
2: ***********************************
